packages/protocol/contracts/common/interfaces/IGoldToken.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.5.13 <0.9.0;
+
+/**
+ * @dev Interface of the ERC20 standard as defined in the EIP. Does not include
+ * the optional functions; to access them see {ERC20Detailed}.
+ */
+interface IGoldToken {
```

Comment:
That was what i'd hoped to do, but unfortunalty interface cannot inherit from other interfaces until pragma ^0.6.

Therefore I had to create this interface because the ERC20 interface we use does not include `mint()` .

Reply:
Ended up moving this contract to only build using `pragma ^0.8` since it's only used by `MintGoldSchedule`